udata/commands/__init__.py

```diff
@@ -30,9 +30,8 @@ def register_commands(manager):
     manager.add_command('clean', Clean())
     manager.add_command('urls', ShowUrls())
 
-    custom_settings = join(os.getcwd(), 'udata.cfg')
-    manager.add_command('serve', Server(port=6666, use_debugger=True, use_reloader=True))
-    # manager.add_command('serve', Server(port=6666, use_debugger=True, use_reloader=True, extra_files=[custom_settings]))
```

Comment:
I kept that until mitsuhiko/werkzeug#662 is fixed.
It seems to be done, maybe we should reintroduce the extra file.